src/app/business/profile/profile.component.ts

```diff
@@ -68,12 +72,51 @@ export class ProfileComponent implements OnInit {
     }
     getProfiles() {
         this.ProfileService.getProfiles().subscribe((res) => {
```

Comment:
what if there is an error on calling this ?

Reply:
@rhsakarpos This has been added to the backlog as #188

Reply:
LG...AB